tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1088,11 +1088,26 @@ static_assert(u8"abc"sv[1] == u8'b');
 static_assert(noexcept(u8"abc"sv));
 #endif // __cpp_char8_t
 
+constexpr bool test_case_range_constructor() {

```

Comment:
This test coverage is great, thanks. Stylistically, I think that this should be defined immediately after `test_case_contiguous_constructor()`, and similarly for the `static_assert`, to keep the test organized. I'll validate and push this change.